ci/scripts/build_iceberg.sh

```diff
@@ -31,6 +31,7 @@ cmake \
     -DICEBERG_BUILD_SHARED=ON \
     ${source_dir}
 cmake --build . --target install
+ctest --output-on-failure -C Debug
```

Comment:
Maybe we want to split testing into a different step on CI but this can be done afterwards